coc/assets/js/coc.js

```diff
@@ -344,7 +344,7 @@ window.coc = ((window, document, $) => {
           xhr.setRequestHeader('X-WP-Nonce', cocVars.nonce);
         },
         success: function(resp) {
-          if(resp.success === true){
+          if(resp !== null && resp.hasOwnProperty('success') && resp.success === true){
```

Comment:
Why checking for `resp.hasOwnProperty('success')` ? :thinking:

Reply:
That was overengineering ;) The property 'success' is always available in the response.
Going to remove it!